docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>Here, there's a slight issue recovering the parameter value. I'm not sure the cause of that. It's close, but the tight 95% credible interval leads me to believe there's something wrong with the implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This is an interesting point. Two things come to my mind:</p><p> </p><ol><li>There could be an issue with the implementation</li><li>We're not using an unbiased estimator.</li></ol><p> </p><p>I tend to think that 2 is much more likely. This is a Bayesian model and, as such, it uses prior distributions. The priors somehow "bias" the asymptotically unbiased maximum likelihood estimator. What's more, I'm not sure how unbiased the ML estimator is.</p><p> </p><p>I tend to think this is more that sort of problem rather than an implementation one. But I'll keep this issue in mind to double check the work done.</p><p> </p><p>And finally, the relative bias is (0.0013 / 0.5)  which is 0.26%. I don't think that's too bad.</p>

Reply:
<p>Sounds good. If 0.26% is good enough, then I'm happy too. </p>